tuna/worker_interface.py

```diff
@@ -768,16 +770,21 @@ def get_compile_jobs(self):
     when the evaluator should stop waiting for jobs to compile"""
     with DbSession() as session:
       try:
+        # pylint: disable=comparison-with-callable
         query = session.query(sqlalchemy_func.count(self.dbt.job_table.id))\
             .filter(self.dbt.job_table.valid == 1)\
             .filter(self.dbt.job_table.session == self.dbt.session.id)\
             .filter(self.dbt.job_table.state.in_(('new', 'started', 'compile_start', 'compiling',
                                                   'compiled')))
+        # pylint: enable=comparison-with-callable
+
         if self.label:
           query = query.filter(self.dbt.job_table.reason == self.label)
         if self.machine.arch:
+          # pylint: disable-next=no-member ; @alex conv_job table appears to have no member 'arch'
```

Comment:
ConvolutionJob appears to not have any members 'arch', 'num_cu'. Could it be some foreign key join stuff going on...?

Reply:
these columns were removed, i guess this pylint version caught it. they should be removed from  the ConvolutionJob class or places that mention it